health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -98,4 +101,133 @@ export const UICustomizations = {
+      
+      console.log("data", data);
+      console.log("additionalDetails", additionalDetails);
+
+      data.params.names = name;
+
+      data.params.phone = phone;
+      
+      data.params.roles = rolesString;
+      data.params.tenantId = Digit.ULBService.getCurrentTenantId();
+      cleanObject(data.params);
+      delete data.params.roleschosen;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)